BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -54,17 +54,21 @@ public actor DefaultTabDataStore: TabDataStore {
             guard let fileURL = windowURLPath(for: uuid, isBackup: false),
                   fileManager.fileExists(atPath: fileURL),
                   let windowData = parseWindowDataFile(fromURL: fileURL) else {
-                logger.log("Failed to open window/tab data for UUID: \(uuid)", level: .fatal, category: .tabs)
+                logger.log("Failed to open window/tab data for UUID: \(uuid)", level: .warning, category: .tabs)
                 throw TabDataError.failedToFetchData
             }
+            logger.log("Successfully fetched window/tab data", level: .debug, category: .tabs)
             return windowData
         } catch {
-            logger.log("Error fetching window data: UUID = \(uuid) Error = \(error)", level: .warning, category: .tabs)
+            logger.log("Error fetching window/tab data for UUID = \(uuid) Error = \(error)", level: .warning, category: .tabs)
             guard let backupURL = windowURLPath(for: uuid, isBackup: true),
                   fileManager.fileExists(atPath: backupURL),
                   let backupWindowData = parseWindowDataFile(fromURL: backupURL) else {
+                let error = "Failed to open backup window/tab data for UUID = \(uuid)"
+                logger.log(error, level: .fatal, category: .tabs)
```

Comment:
@OrlaM Does it make sense to log another fatal error here when we also fail to get data from the backup file?

Reply:
@ih-codes the change is outdated, do you want to resolve this?

Reply:
Yeah I'll resolve. I just pushed up significant additions to add more logging, I'm going to re-request a review from everyone. 👍